src/battle_main.c

```diff
@@ -3115,11 +3115,15 @@ static void BattleStartClearSetData(void)
     }
 }
 
+#define UNPACK_VOLATILE_BATON_PASSABLES(_enum, _type, _fieldNameBitSize, _flags, ...) INVOKE(UNPACK_VOLATILE_BATON_PASSABLES_, _flags, UNPACK(_fieldNameBitSize))
+#define UNPACK_VOLATILE_BATON_PASSABLES_(_flags, _fieldName, ...) if (_flags & V_BATON_PASSABLE) gBattleMons[battler].volatiles._fieldName = volatilesCopy._fieldName;
```

Comment:
Need parens (`&` has higher precedence than `|`).
```suggestion
#define UNPACK_VOLATILE_BATON_PASSABLES_(_flags, _fieldName, ...) if ((_flags) & V_BATON_PASSABLE) gBattleMons[battler].volatiles._fieldName = volatilesCopy._fieldName;
```